testsuite/E12.py

```diff
@@ -1,22 +1,22 @@
 #: E121
-print "E121", (
```

Comment:
If parentheses are important for these tests (which is likely), maybe I could replace the

    print expression

with

    raise expression

or

    assert expression